README.md

```diff
@@ -355,14 +348,14 @@ Storyblok.getAll('cdn/stories', {
 
 ```javascript
 Storyblok.post('spaces/<YOUR_SPACE_ID>/stories', {
-	story: { name: 'xy', slug: 'xy' },
+  story: { name: 'xy', slug: 'xy' },
 })
-	.then((response) => {
-		console.log(response)
-	})
-	.catch((error) => {
-		console.log(error)
-	})
+  .then((response) => {
+    console.log(response)
+  })
+  .catch((error) => {
+    console.log(error)
```

Comment:
Suggestion: `console.error` would be better practice than `console.log` in a `catch` callback.

Reply:
This got merged @manuelschroederdev